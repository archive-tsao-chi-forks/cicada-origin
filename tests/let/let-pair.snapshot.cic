def x: (Nat * Nat) =
  let one = add1(zero)
  let two = add1(one)
  let three = add1(two)
  cons(one, two)

x
