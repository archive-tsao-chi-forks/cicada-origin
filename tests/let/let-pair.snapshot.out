the (_: Nat * Nat) cons(1, 2)

